console/src/components/MapCore/MapCore.module.less

```diff
@@ -74,4 +74,5 @@
 // we don't have control over and can't get back to our app from.
 :global(.mapboxgl-control-container) * {
 	pointer-events: none;
-}
+	display: none;
```

Comment:
check if this change is still needed after downgrading to version 1.x

Reply:
This change was not needed and removed after the rollback to version 1.13.0